catroid/src/org/catrobat/catroid/content/Sprite.java

```diff
@@ -64,7 +65,7 @@
 	private ArrayList<UserBrick> userBricks;
 	private transient int newUserBrickNext = 1;
 
-	private Integer id;
+	private int id = ProjectManager.getInstance().getNewId();
```

Comment:
For consistency.
If a new Sprite is created, the Sprite gets a new id.
If the Sprite is loaded from the XML, the newly received id at this line will be replaced by the id from the xml by xstream.